### PR TITLE
Add test file by koguma

### PR DESCRIPTION
**Pull request by aim-oguma**
* TEST commit for all infra members in Philippines.

### CODE REVIEW

File: koguma [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ruby -e "puts 'This is test: foo bar'"
+
```
Comment (agsi-mmilitante) [inline koguma]: https://i.gyazo.com/ad90459ebd51d65af30b11dedf58b496.png

File: koguma [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+for (( i = 1 ; i <= 3 ; i++ ))
+do
+  echo "$i times speak[ Magandang hapon po ]"
```
Comment (jinellevicuna) [inline koguma]: Magandang hapon din po.

File: koguma [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+done
```
Comment (ags-mc) [inline koguma]: Comment done! :dancer:

Comment (ags-mak) [inline koguma]: test comment

File: koguma (hunk 2 above)
Comment (aim-oguma) [inline koguma]: :)
Magandang hapon din po.

Comment (bedista) [inline koguma]: konnichiwa

**Pull request merged**